postgis/postgis.sql.in

```diff
@@ -6982,7 +6982,7 @@ CREATE OR REPLACE FUNCTION ST_3DLineInterpolatePoint(geometry, float8)
 -- ST_RemoveIrrelevantPointsForView
 -----------------------------------------------------------------------
 -- Availability: 3.5.0
-CREATE OR REPLACE FUNCTION ST_RemoveIrrelevantPointsForView(geometry, box2d)
+CREATE OR REPLACE FUNCTION ST_RemoveIrrelevantPointsForView(geometry, box2d, boolean default false)
```

Comment:
Beware about changing signatures. While we're in development mode it's not too much of a problem but it's still nice to handle that signatures need to be removed or will remain in the database and create a problem later.

See postgis/postgis_after_upgrade.sql

Reply:
Thank you for that hint. I carefully changed this signature and left the new variable optional with default false which makes the old signature IMO obsolete. For the current case (still in developing mode) - would it sufficient to add  _SELECT _postgis_drop_function_by_signature('ST_RemoveIrrelevantPointsForView(geometry, box2d)');_ in postgis/postgis_after_upgrade.sql?

Reply:
@strk: The removal of the old signature is now handled in _postgis/postgis_after_upgrade.sql_ as suggested. Tests were successful.